website/snippets/warehouse-setups-cloud-callout.md

```diff
@@ -1,3 +1,3 @@
-:::info `profiles.yml` file is for dbt Core users only
-If you're using <Constant name="dbt_platform" />, you don't need to create a `profiles.yml` file. This file is only for dbt Core users. To connect your data platform to <Constant name="cloud" />, refer to [About data platforms](/docs/cloud/connect-data-platform/about-connections). 
+:::info `profiles.yml` file is for local development with dbt Core and dbt fusion only
```

Comment:
No a blocker but since this will sort of be a header we could shorten it
```suggestion
:::info `profiles.yml` file is for dbt Core and dbt fusion only
```